src/throttler.guard.ts

```diff
@@ -122,7 +123,7 @@ export class ThrottlerGuard implements CanActivate {
    * @throws ThrottlerException
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  protected throwThrottlingException(context: ExecutionContext): void {
+  protected throwThrottlingException(context: ExecutionContext, info: ThrottlingInfo): void {
```

Comment:
I guess we can improve the type def of `throwThrottlingException`

```suggestion
  protected throwThrottlingException(context: ExecutionContext, info: ThrottlingInfo): never {
```

[isn't `void`](https://thehotcode.com/typescript-difference-void-never/#never) because that method must always throw an error